unit2-java/lesson06.rst

```diff
@@ -4,14 +4,18 @@ Unit 2 (Java) - Lesson 6 Learning Objectives
 Goals
 -----
 
+- Understand the role of inheritance in enable effective code resuse.
+- Effectively use Java's inheritance tools to build clean, reusable code.
+
 Objectives
 ----------
 
 - Use the ``extends`` keyword to allow one class to inherit from another
-- Explain which fields and methods are inherited by a class extension, based on Java’s rules and taking into account access modifiers
+- Explain which fields and methods are inherited by a class extension, based on Java's rules and taking into account access modifiers
```

Comment:
Just a general "why-am-I-finally-noticing-this" comment - to accommodate screen readers, we should convert the lists from ul to ol.

Reply:
`ul` elements aren't incompatible with screen readers in and of themselves. The crucial point is to use `ol` and `ul` in the proper contexts. For example, for a table of contents with numbered chapters and sections `ol` should be used. The goals and objectives listed here aren't in any inherent or particular order, so I think `ul` is fine for this purpose.

Reply:
OK.